### DISCUSSION TITLE
Problem using Relation Reference with Qfield.

### DISCUSSION THREAD

**Discussion by mahmoodi-source** [opengisch/QField#5210] [Q&A]
Hi everyone,
I am working on a QGIS project to upload on Qfield, and in this project, I have created a 1:n relation between a table with a point geometry that represents trees and another table without geometry that represents the conditions of the trees. But when I choose the "Relation Reference" widget type in the attribute form of the main table, the "Relation" tab is visible. However, the "Relation" dropdown is empty, which means that the project hasn't recognized the relation I have created. Is there anyway to resolve this?
![relation](https://github.com/opengisch/QField/assets/144428014/296d13a1-df79-42bd-849f-814a748ddd16)
![attribute form](https://github.com/opengisch/QField/assets/144428014/6e6af7db-3f2c-4fb2-8078-91de483b01a9)
![Uploading Screenshot_20240425_103910.jpg…]()

**Comment by m-kuhn** [comment]
Have a look at the documentation (also upstream on the link to qgis) https://docs.qfield.org/how-to/attributes-form/#configure-a-relation-widget
It is quite well covered there
